mnefun/_mnefun.py

```diff
@@ -1875,7 +1897,10 @@ def plot_raw_psd(p, subjects, run_indices=None, tmin=0., fmin=2, n_fft=2048):
                 warnings.warn('Unable to find %s data file.' % file_type)
             with warnings.catch_warnings(record=True):
                 raw = Raw(fname, preload=True, allow_maxshield=True)
-            fmax = p.lp_cut if file_type == 'pca' else raw.info['lowpass'] + 50
+            if file_type == 'pca':
+                fmax = p.lp_cut
+            else:
+                fmax = raw.info['lowpass'] + 50
```

Comment:
No, the line used to have some parens that made it > 80 char, and this was the other fix I had done. Doesn't really matter much either way.